mist-lib/src/signal-transduction/mist3-sig-trans-2017.csv

```diff
@@ -0,0 +1,466 @@
+id,domain_name,superfamily,description,function,two_components,sig_trans_specific,source,pmid
```

Comment:
What about placing this file in a subdirectory called rules?

Reply:
ok